hw/ip/i2c/doc/dv_plan/index.md

```diff
@@ -54,7 +54,7 @@ which provides the ability to drive and independently monitor random traffic via
 TL host interface into I2C device.
 
 ### I2C agent
-I2C agent is configured to work device mode and implemented as [reactive slave](https://www.verilab.com/files/mastering_reactive_slaves.pdf)
+I2C agent is configured to work device mode and implemented as [reactive target](https://www.verilab.com/files/mastering_reactive_slaves.pdf)
```

Comment:
I'm not sure about this: "reactive slave" is a UVM technical term, which I don't think currently has a synonym. Definitely "reactive target" doesn't have the desired meaning (or any meaning, I don't think). If we want to avoid "slave", we could expand to something like:

```
The I2C agent is configured to work in device mode and reacts to requests from the DUT (see [this paper](https://www.verilab.com/files/mastering_reactive_slaves.pdf) for details of the framework).
```

Reply:
I think these can also be referred to as "reactive agents", maybe that could work here instead (this is how I refer to all of the agents used in the Ibex testbench).